packages/pipeline/src/Stage.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { forwardRef, ReactNode, Ref, ReactElement } from 'react';
+import PropTypes from 'prop-types';
+import mergeRefs from 'react-merge-refs';
+import { useInView } from 'react-intersection-observer';
+
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import {
+  getRootStyle,
+  getChildStyle,
+  getChevronStyle,
+  Size,
+  Variant,
+  layout,
+  colors,
+  stageAttr,
+  stageChevronAttr,
+} from './styles';
+
+export interface StateForStyles {
```

Comment:
Doesn't seem like we need to export this?

Reply:
Resolved by 9f54a5e